utilities/unittests/test_bitwarden.py

```
@@ -21,6 +21,15 @@
 
 from utilities.exceptions import MissingEnvironmentVariableError
 
+_original_timeout_sampler_init = TimeoutSampler.__init__
+
+
+def _short_timeout_init(self, *args, **kwargs):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_short_timeout_init`

Add return type annotation: `None`

(ANN202)